driver.go

```diff
@@ -229,70 +256,71 @@ func iso8859(s string) (time.Time, error) {
 	return time.Parse(time.RFC3339, s)
 }
 
-// PreCreateCheck allows for pre-create operations to make sure a driver is ready for creation
+// PreCreateCheck allows for pre-create operations to make sure a driver is
+// ready for creation
 func (d *Driver) PreCreateCheck() error {
-	client, err := d.client()
+	c, err := d.client()
 	if err != nil {
 		return err
 	}
 
-	// TODO find a better "Ping" method
-	_, err = client.CountMachines()
+	_, err = c.Ping(context.Background())
```

Comment:
Nice! :metal:

Reply:
I liked adding this as well. Despite the fact that we don't technically need to be authenticated it'll at least test that the connection to CloudAPI works properly.